packages/bundler-plugin-core/src/index.ts

```diff
@@ -161,6 +164,31 @@ export function sentryUnpluginFactory({
       })
     );
 
+    if (options.bundleSizeOptimizations) {
+      const { bundleSizeOptimizations } = options;
+      const values: SentrySDKBuildFlags = {};
```

Comment:
l: I would maybe name this something like stringReplacementValues or something like that.